src/main/java/core/basesyntax/fruitshop/report/generator/ReportGenerator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.fruitshop.report.generator;
+
+import java.util.Map;
+
+public interface ReportGenerator {
+    void generateReport(String toFile, Map<String, Integer> storage);
+}
```

Comment:
```suggestion
public interface ReportGenerator {
    String generateReport(); // there is no need to pass storage, you can access it from the method
}

// another service

public interface FileWriter {
    void write(String pathToFile, String content);
}
```
let's split this service into 2: currently it does 2 separate things
generating a report is one responsibility, and writing to file is another
we will be able to reuse them independently later